relay-server/src/actors/store.rs

```diff
@@ -439,6 +446,54 @@ impl StoreForwarder {
         );
         Ok(())
     }
+
+    fn produce_replay_recording_chunks(
+        &self,
+        replay_id: EventId,
+        project_id: ProjectId,
+        item: &Item,
+    ) -> Result<ChunkedAttachment, StoreError> {
+        let id = Uuid::new_v4().to_string();
+
+        let mut chunk_index = 0;
+        let mut offset = 0;
+        let payload = item.payload();
+        let size = item.len();
+
+        // This skips chunks for empty replay recordings. The consumer does not require chunks for
+        // empty replay recordings. `chunks` will be `0` in this case.
+        while offset < size {
```

Comment:
these bits are copied from the attachment chunks produce function.

Reply:
Thanks for calling this out. For now this is totally fine, once we have a third use case, we can still generalize the chunking implementation.